drivers/sensor/bme680/bme680.c

```diff
@@ -414,6 +414,8 @@ static int bme680_init(const struct device *dev)
 
 	data->i2c_slave_addr = DT_INST_REG_ADDR(0);
 
+	k_msleep(2); /* Start up delay */
```

Comment:
@anangl Isn't there a way to do asynchronous init of devices? I don't think it is. For motion sensor we did some workaround but it would be great to add some common thing or simply define that until init is completed any API called by application will return with -EBUSY (this is what we did).

Reply:
I don't know of any standard way of doing this. And I agree, it would be nice to have such common mechanism.

Reply:
Discussed offline with @anangl. We decided to add one delay in `board.c` file to give all sensors time to settle. Because of that proper initialization chain is required which would be asserted within this file.

Reply:
This commit would be dropped.